### ISSUE TITLE
binding is shared

### ISSUE THREAD

**Issue by alexvasseur** [avasseur-pivotal/postgresql-cf-service-broker#7]
unlike p-mysql the binding info is shared and not unique across binding with multi user and grand mechanics
this is not ideal for revoke - but at the same time note that an app granted access one day could create another user to keep that user unless privileges are not allowing this specific scenario which is more work

see attached - after 1 service + 1 binding

<img width="437" alt="screen shot 2017-03-16 at 00 38 52" src="https://cloud.githubusercontent.com/assets/5575656/23975468/373aaa6a-09e1-11e7-8cf0-a008262ba473.png">
<img width="578" alt="screen shot 2017-03-16 at 00 38 26" src="https://cloud.githubusercontent.com/assets/5575656/23975469/375385da-09e1-11e7-8fd2-db7a67c3ac70.png">

**Comment by alexvasseur** [comment]
To be confirmed
2 sb looks like this:

```
postgres://0a10ba1b-5c00-4434-89b8-498ecb66d0d7:5fm0s88197krjuq8fl9f51a32e@XXX:5432/0a10ba1b-5c00-4434-89b8-498ecb66d0d7"

postgres://0a10ba1b-5c00-4434-89b8-498ecb66d0d7:i5h1bf3aohaibf17l4k9c5gukd@XXX:5432/0a10ba1b-5c00-4434-89b8-498ecb66d0d7

```

**Comment by alexvasseur** [comment]
may be an unbind will also drop the db schema for the other bound service - which is really bad

**Comment by alexvasseur** [comment]
fixed in 3.0
